src/ablog/__init__.py

```diff
@@ -118,7 +119,13 @@ def builder_inited(app):
     theme = app.builder.theme
     loaders = app.builder.templates.loaders
     templatepathlen = app.builder.templates.templatepathlen
-    if theme.get_config("ablog", "inject_templates_after_theme", False):
+    try:
+        # Modern Sphinx now errors instead of returning the default if there is not a value
+        # in any of the config files.
```

Comment:
I think it is worse than that, new version of sphinx always return the default. It never looks in config files for things that are not `theme`, or `options`.

Reply:
Well, the code will work but the comment is not technically correct. Sphinx 7.3.0 to 7.3.6 will indeed error. 

But sphinx 7.3.7 and above will just ignore the config and return the default value (False in this case).

https://github.com/sphinx-doc/sphinx/blob/de4ac2fbdebae529a973910e96794645158f0401/sphinx/theming.py#L123-L134

Basically sphinx removed support for custom sections.

Reply:
In that case, I will need to work out how to handle this in a manner that sphinx likes.

But for now, I will just ignore this and pretend it didn't break.